src/Helpers/LinkRenderer.php

```diff
@@ -23,32 +25,36 @@ public function render(AbstractInline $inline, ElementRendererInterface $htmlRen
             throw new \InvalidArgumentException('Incompatible inline type: ' . get_class($inline));
         }
 
-        $attrs = array();
+        $attributes = [
+            'href' => $this->getHref($inline->getUrl())
+        ];
 
-        if (isset($inline->attributes['title'])) {
-            $attrs['title'] = $htmlRenderer->escape($inline->data['title'], true);
+        if (isset($inline->attributes['title']) and strlen($inline->attributes['title']) > 0) {
+            $attributes['title'] = $htmlRenderer->escape($inline->data['title'], true);
         }
 
-        $url = $inline->getUrl();
-        if ($this->isExternalUrl($url)) {
-            $attrs['class'] = 'external-link';
-            $attrs['target'] = '_blank';
-            $attrs['rel'] = 'noreferrer noopener';
+        if (static::isExternalUrl($inline->getUrl())) {
+            $attributes['class'] = 'external-link';
+            $attributes['target'] = '_blank';
+            $attributes['rel'] = 'noreferrer noopener';
         }
-        elseif (strpos($url, '#') === 0) {
```

Comment:
This condition is needed to fix `#achor` urls

Reply:
The logic is split into the `getHref` method as the value of `href` needs to be sat regardless.